rules/windows/file/file_event/file_event_win_creation_configuration_rats.yml

```
@@ -0,0 +1,33 @@
+title: File Creation Related To RAT Clients
+id: 2f3039c8-e8fe-43a9-b5cf-dcd424a2522d
+status: test
```

```suggestion
status: experimental
```
See https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq-rule-convention.md#status